superset-frontend/src/dashboard/components/FiltersBadge/FilterIndicator/index.tsx

```diff
@@ -40,6 +40,7 @@ const FilterIndicator: FC<IndicatorProps> = ({
   const resultValue = getFilterValueForDisplay(value);
   return (
     <FilterItem
+      role="button"
```

Comment:
Similar to other comments, the button role could probably be added conditionally